### PR TITLE
Enabling dht log in the benchmark

### PR DESCRIPTION
**Pull request by sim590**
@aberaud, if you can merge this as it is pretty straightforward.

I want to specify that I've had to split `tools_common.h` for linking successfully with the cython generated shared object. Otherwise, some code that wouldn't be relevant to link with, like readline, would fail to link if not compiled with `-lreadline`. But cython module just doesn't need that, hence the split.

### CODE REVIEW

File: python/setup.py.in [hunk 1]
```diff
@@ -48,7 +48,7 @@ setup(name="opendht",
       ext_modules = cythonize(Extension(
           "opendht",
           ["@CURRENT_SOURCE_DIR@/opendht.pyx"],
-          include_dirs = ['@PROJECT_SOURCE_DIR@/include'],
+          include_dirs = ['@PROJECT_SOURCE_DIR@/include', '@PROJECT_SOURCE_DIR@/tools'],
```
Comment (aberaud) [inline python/setup.py.in]: If log.h may be used by other components, let's move it to 'include'.
I'm not too comfortable with the python layer depending on something in 'tools'.

File: tools/log.h [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301 USA.
+ */
+
+#pragma once
+
+#include <opendht.h>
```
Comment (aberaud) [inline tools/log.h]: include the minimum, opendht/dhtrunner.h

File: tools/log.h [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+#include <iostream>
+#include <fstream>
+
```
Comment (aberaud) [inline tools/log.h]: add namespace dht::log::

File: tools/tools_common.h [hunk 4]
```diff
@@ -2,6 +2,7 @@
  *  Copyright (C) 2014-2016 Savoir-faire Linux Inc.
  *
  *  Author: Adrien Béraud <adrien.beraud@savoirfairelinux.com>
+ *          Simon Désaulniers <sim.desaulniers@gmail.com>
```
Comment (aberaud) [inline tools/tools_common.h]: don't add copyright if you didn't contributed to a file

File: python/setup.py.in (hunk 1 above)
Comment (sim590) [inline python/setup.py.in]: Yes, you're right! Will do.

**Comment by sim590** [comment]
@aberaud, I have addressed your suggestions. It's much better now. Tell me what you think.

**Pull request merged**